openmc/settings.py

```diff
@@ -91,6 +91,8 @@ class Settings(object):
         Indicate whether survival biasing is to be used
     weight : float
         Weight cutoff below which particle undergo Russian roulette
+    energy : float
+        Energy cutoff below which particle will be killed
```

Comment:
I think on the Python side, we should just have a `Settings.cutoff` attribute that is a dictionary, so e.g., you could set
```Python
Settings.cutoff = {'energy': 1e-11}
```
Are you open to implementing that?